### PR TITLE
Update deep_feedforward_networks.tex

### PR DESCRIPTION
**Pull request by jaysoon**
这句话的原译文不流畅，One of these algorithmic changes was the replacement of mean squared error with the cross-entropy family of loss functions.

### CODE REVIEW

**Comment by KevinLee1110** [comment]
有道理，多谢~
